### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by TheiLLeniumStudios**
**Describe Pull request**
Ignore the status checks. The errors will be gone once this is merged

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

**Comment by IdrisDose** [comment]
@TheiLLeniumStudios bump plz fix checks ;)

**Comment by TheiLLeniumStudios** [comment]
@IdrisDose All checks are passing now

File: client/dead.lua [hunk 1]
```diff
@@ -80,7 +80,7 @@ function DeathTimer()
     end
 end
 
-local function DrawTxt(x, y, width, height, scale, text, r, g, b, a, outline)
+local function DrawTxt(x, y, width, height, scale, text, r, g, b, a, _)
```
Comment (BerkieBb) [inline client/dead.lua:83]: Instead of makong it an underscore, just remove it

File: client/job.lua [hunk 2]
```diff
@@ -390,7 +368,7 @@ end
 
 RegisterNetEvent('qb-ambulancejob:elevator_roof', function()
     local ped = PlayerPedId()
-    for k, v in pairs(Config.Locations["roof"])do
+    for k, _ in pairs(Config.Locations["roof"])do
```
Comment (BerkieBb) [inline client/job.lua:371]: You can remove this underscore completely

File: client/job.lua [hunk 3]
```diff
@@ -410,7 +388,7 @@ end)
 
 RegisterNetEvent('qb-ambulancejob:elevator_main', function()
     local ped = PlayerPedId()
-    for k, v in pairs(Config.Locations["main"])do
+    for k, _ in pairs(Config.Locations["main"])do
```
Comment (BerkieBb) [inline client/job.lua:391]: You can remove this underscore completely

File: client/main.lua [hunk 4]
```diff
@@ -54,7 +53,7 @@ local function GetAvailableBed(bedId)
     local pos = GetEntityCoords(PlayerPedId())
     local retval = nil
     if bedId == nil then
-        for k, v in pairs(Config.Locations["beds"]) do
+        for k, _ in pairs(Config.Locations["beds"]) do
```
Comment (BerkieBb) [inline client/main.lua:56]: You can remove this underscore completely

File: client/main.lua [hunk 5]
```diff
@@ -128,7 +127,7 @@ local function SetClosestBed()
     local pos = GetEntityCoords(PlayerPedId(), true)
     local current = nil
     local dist = nil
-    for k, v in pairs(Config.Locations["beds"]) do
+    for k, _ in pairs(Config.Locations["beds"]) do
```
Comment (BerkieBb) [inline client/main.lua:130]: You can remove this underscore completely

File: client/main.lua [hunk 6]
```diff
@@ -319,7 +318,7 @@ end
 local function IsInDamageList(damage)
     local retval = false
     if CurrentDamageList then
-        for k, v in pairs(CurrentDamageList) do
+        for k, _ in pairs(CurrentDamageList) do
```
Comment (BerkieBb) [inline client/main.lua:321]: You can remove this underscore completely

File: server/main.lua [hunk 7]
```diff
@@ -213,7 +213,7 @@ QBCore.Functions.CreateCallback('hospital:GetPlayerStatus', function(source, cb,
 			if (PlayerInjuries[Player.PlayerData.source].isBleeding > 0) then
 				injuries["BLEED"] = PlayerInjuries[Player.PlayerData.source].isBleeding
 			end
-			for k, v in pairs(PlayerInjuries[Player.PlayerData.source].limbs) do
+			for k, _ in pairs(PlayerInjuries[Player.PlayerData.source].limbs) do
```
Comment (BerkieBb) [inline server/main.lua:216]: You can remove this underscore completely

File: server/main.lua [hunk 8]
```diff
@@ -241,18 +241,19 @@ end)
...
-    for k,v in pairs(players) do
+    for _, v in pairs(players) do
         if v.PlayerData.job.name == 'ambulance' and v.PlayerData.job.onduty then
             TriggerClientEvent('hospital:client:ambulanceAlert', v.PlayerData.source, coords, message)
         end
     end
 end)
 
-QBCore.Commands.Add("status", Lang:t('info.check_health'), {}, false, function(source, args)
+QBCore.Commands.Add("status", Lang:t('info.check_health'), {}, false, function(source, _)
```
Comment (BerkieBb) [inline server/main.lua:256]: You can remove this underscore completely

File: server/main.lua [hunk 9]
```diff
@@ -262,7 +263,7 @@ QBCore.Commands.Add("status", Lang:t('info.check_health'), {}, false, function(s
 	end
 end)
 
-QBCore.Commands.Add("heal", Lang:t('info.heal_player'), {}, false, function(source, args)
+QBCore.Commands.Add("heal", Lang:t('info.heal_player'), {}, false, function(source, _)
```
Comment (BerkieBb) [inline server/main.lua:266]: You can remove this underscore completely

File: server/main.lua [hunk 10]
```diff
@@ -272,7 +273,7 @@ QBCore.Commands.Add("heal", Lang:t('info.heal_player'), {}, false, function(sour
 	end
 end)
 
-QBCore.Commands.Add("revivep", Lang:t('info.revive_player'), {}, false, function(source, args)
+QBCore.Commands.Add("revivep", Lang:t('info.revive_player'), {}, false, function(source, _)
```
Comment (BerkieBb) [inline server/main.lua:276]: You can remove this underscore completely

**Review by BerkieBb — CHANGES_REQUESTED** [review]

**Pull request merged**